docs/source/apps/max_sharpe_ratio_optimization.ipynb

```diff
@@ -1271,9 +1271,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3 (ipykernel)",
+   "display_name": "Python 3",
    "language": "python",
-   "name": "python"
+   "name": "python3"
```

Comment:
does this affect anything?

Reply:
Not that I'm aware of. The standard/default metadata name for all the other notebooks is "python3" hence why I updated it for consistency. Shouldn't change much as this will get rewritten with your specific kernel name when running the notebook locally.